modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/process-metrics/performance-by-step-card/PerformanceByStepCardBody.es.js

```diff
@@ -47,16 +47,18 @@ function Body({items, totalCount}) {
 	);
 }
 
-function Footer({processId, timeRange, totalCount}) {
+function Footer({processId, processVersion, timeRange, totalCount}) {
 	const {defaultDelta} = useContext(AppContext);
 	const filters = {};
 
 	const {dateEnd, dateStart, key} = timeRange;
+
 	if (dateEnd && dateStart && key) {
 		filters.dateEnd = dateEnd;
 		filters.dateStart = dateStart;
 		filters.timeRange = [key];
 	}
+	filters.processVersion = processVersion;
```

Comment:
@naraandrade new line before `filters.processVersion = processVersion;`? I understood why you didn't use here but it seems weird because of the if's bracket. 

Thx

Reply:
Yes, it seems a little weird... I will add the line, thanks :)